ChiModules/KEigenvalueSolver/k_eigenvalue_solver.h

```diff
@@ -16,7 +16,8 @@ class Solver : public LinearBoltzmann::Solver
   size_t source_event_tag;
 
 public:
-  int num_precursors = 0;
+  int num_precursors;
```

Comment:
Please change `int num_precursors;` to `size_t num_precursors;`.

Reply:
Done.